src/web/toga_web/widgets/button.py

```diff
@@ -1,26 +1,29 @@
+
+from toga_web.libs import create_element
+
 from .base import Widget
 
 
 class Button(Widget):
-    def __html__(self):
-        return """
-            <button id="toga_{id}" class="toga button btn-block" style="{style}">
-            {text}
-            </button>
-        """.format(
-            id=self.interface.id,
-            text=self.interface.text,
-            style='',
+    def create(self):
+        self.native = create_element(
+            "button",
+            id=f"toga_{self.interface.id}",
+            classes=["button", "btn-block"],
+            style=self.interface.style.__css__(),
```

Comment:
Since `id` and `style` will be the same for all widgets, this should be factored out.